prisma/schema.prisma

```diff
@@ -0,0 +1,57 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id                 String         @id(map: "pk_User") @db.Uuid
+  email              String         @unique
+  password           String
+  resetPasswordToken String?
+  isVerified         Boolean        @default(false)
+  roles              RolesOnUsers[]
+  createdAt          DateTime       @default(now())
+  updatedAt          DateTime       @default(now())
+
+  @@index([isVerified])
+  @@map("users")
+}
+
+model Role {
```

Comment:
no need to make it complex from the beginning. remove Role and RolesOnUsers models, and just use the enum RolesEnum
  roles           UserRolesEnum[] @default([CUSTOMER])